examples/property_estimator/start_client.py

```diff
@@ -8,22 +8,43 @@
 from propertyestimator.utils import get_data_filename, setup_timestamp_logging
 
 
-def submit_calculation_to_server():
+def request_estimate_blocking():
```

Comment:
Suggest `compute_estimate()` might be simpler and clearer.
The blocking nature of this call is implied because we're asking for the estimate to be computed (now), as well as documented.